### PR TITLE
Upgrade to RN 0.50.4

### PR DESCRIPTION
**Pull request by csduarte**
Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

When filling in a section please remove the help text and the above text.

#### Summary
[A brief description of what this pull request does.]

#### Ticket Link
[Please link the GitHub issue or Jira ticket this PR addresses.]

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates

#### Device Information
This PR was tested on: [Device name(s), OS version(s)] 

#### Screenshots
[If the PR includes UI changes, include screenshots (for both iOS and Android if possible).]

### CODE REVIEW

File: app/components/channel_drawer/channels_list/filtered_list/filtered_list.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ import {displayUsername} from 'mattermost-redux/utils/user_utils';
 
 import ChannelDrawerItem from 'app/components/channel_drawer/channels_list/channel_item';
 
+const VIEWABILITY_CONFIG = {
```
Comment (enahum) [inline app/components/channel_drawer/channels_list/filtered_list/filtered_list.js]: can we make this in a constant file? I see this used in three different places

**Comment by enahum** [comment]
Also can you make sure you update `react-native-svg` and replace in the makefile in the post install script the line for that library with this one

```
@cd ./node_modules/react-native-svg/ios && rm -rf PerformanceBezier QuartzBookPack && yarn run postinstall
```

thanks @jarredwitt

File: app/components/channel_drawer/channels_list/filtered_list/filtered_list.js (hunk 1 above)
Comment (jarredwitt) [reply app/components/channel_drawer/channels_list/filtered_list/filtered_list.js]: Will do.

**Comment by enahum** [comment]
@jarredwitt needs rebase and still need the changes that we discussed

**Pull request merged**